src/proof.rs

```diff
@@ -541,8 +541,6 @@ impl<'a, 'b> ProofTreePrinterImpl<'a, 'b> {
                 }
             }
         }
-        proof_ordered_hyps.append(&mut proof_ordered);
-        proof_ordered = proof_ordered_hyps;
```

Comment:
Essentials were previously added to the list of statements in the parens. This is removed.